### PR TITLE
[LEARNMORE] deploy with docker

### PR DESCRIPTION
**Pull request by isabelizimm**
Extended demo on how to deploy with docker!  

**Goals for readers**: understand what is needed to build and run Dockerfiles with vetiver.

Do we need more background on what Docker is/where it is accepted?

### CODE REVIEW

File: learn-more/deploy-with-docker.qmd [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+
+### Getting started with Docker and R
+
+- [Docker for the UseR](https://github.com/noamross/nyhackr-docker-talk)
+- [Enough Docker to be Dangerous](https://seankross.com/2017/09/17/Enough-Docker-to-be-Dangerous.html)
+- [Docker tutorial for reproducible research](https://jsta.github.io/r-docker-tutorial/)
+
+### Getting started with Docker and Python
+
+- tktk
```
Comment (juliasilge) [inline learn-more/deploy-with-docker.qmd:260]: I do think pointing to some resources on getting started with Docker would be good for this article. Can you add 2 or 3 that you personally think are helpful for folks getting started?

**Comment by juliasilge** [comment]
In my edits above ⬆️ I tried to make it more clear what was the command line and what was R/Python. 

This will be so great to publish! 🙌

File: learn-more/deploy-with-docker.qmd (hunk 1 above)
Comment (juliasilge) [reply learn-more/deploy-with-docker.qmd:260]: Let's add this here as well: https://solutions.rstudio.com/environments/docker/

Maybe we don't separate out into R and Python, but have one list that includes both? We still should have a couple that are more for a Python user.

Comment (isabelizimm) [reply learn-more/deploy-with-docker.qmd:260]: Perfect! I can add that one in along with a few Python resources. 👍

**Pull request merged**